src/Particular.CodeRules/Particular.ruleset

```diff
@@ -230,4 +230,54 @@
     <Rule Id="CA5403" Action="None" />             <!-- Do not hard-code certificate -->
     <Rule Id="CA9999" Action="None" />             <!-- Analyzer version mismatch -->
   </Rules>
-</RuleSet>
\ No newline at end of file
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <!-- rules we may want to enforce eventually -->
+    <Rule Id="SA0001" Action="None" />             <!-- XML comment analysis disabled -->
+    <Rule Id="SA1005" Action="None" />             <!-- Single line comments should begin with single space -->
+    <Rule Id="SA1009" Action="None" />             <!-- Closing parenthesis should be spaced correctly -->
+    <Rule Id="SA1012" Action="None" />             <!-- Opening braces should be spaced correctly -->
+    <Rule Id="SA1013" Action="None" />             <!-- Closing braces should be spaced correctly -->
+    <Rule Id="SA1028" Action="None" />             <!-- Code should not contain trailing whitespace -->
+    <Rule Id="SA1121" Action="None" />             <!-- Use built-in type alias -->
+    <Rule Id="SA1127" Action="None" />             <!-- Generic type constraints should be on their own line -->
+    <Rule Id="SA1128" Action="None" />             <!-- Put constructor initializers on their own line -->
+    <Rule Id="SA1135" Action="None" />             <!-- Using directives should be qualified -->
+    <Rule Id="SA1201" Action="None" />             <!-- Elements should appear in the correct order -->
+    <Rule Id="SA1202" Action="None" />             <!-- Elements should be ordered by access -->
+    <Rule Id="SA1203" Action="None" />             <!-- Constants should appear before fields -->
+    <Rule Id="SA1204" Action="None" />             <!-- Static elements should appear before instance elements -->
+    <Rule Id="SA1205" Action="None" />             <!-- Partial elements should declare access -->
+    <Rule Id="SA1208" Action="None" />             <!-- System using directives should be placed before other using directives -->
+    <Rule Id="SA1214" Action="None" />             <!-- Readonly fields should appear before non-readonly fields -->
+    <Rule Id="SA1302" Action="None" />             <!-- Interface names should begin with I -->
+    <Rule Id="SA1306" Action="None" />             <!-- Field names should begin with lower-case letter -->
+    <Rule Id="SA1307" Action="None" />             <!-- Accessible fields should begin with upper-case letter -->
+    <Rule Id="SA1311" Action="None" />             <!-- Static readonly fields should begin with upper-case letter -->
+    <Rule Id="SA1314" Action="None" />             <!-- Type parameter names should begin with T -->
+    <Rule Id="SA1400" Action="None" />             <!-- Access modifier should be declared -->
```

Comment:
I know this is not something we currently have but oh boy this would make things so much more readable and I'm pretty sure the only person that would argue "this is noise" has long left the company 🥇